### ISSUE TITLE
No verification for 'eHerkenning'

### ISSUE THREAD

**Issue by SylvesterMachielse** [Evidos/SignhostClientLibrary#37]
The endpoint documentation contains an entry for 'eHerkenning':

[https://evidos.github.io/endpoints/##/definitions/Verification](https://evidos.github.io/endpoints/##/definitions/Verification)

All the verifications have a corresponding class in `Signhost.APIClient.Rest.DataObjects`, but there is no class for `eHerkenning`.

**Comment by WouterHD** [comment]
You are correct, there is indeed no `EHerkenningVerification` object. It is however still possible to add an 'eHerkenning' verification to your transaction by using the `SignHostApiClient.RegisterVerification<T>();` method. You can make a class that inherits the `IVerification` with 'eHerkenning' as a value for the `Type` property. 

This can be used to add verifications that are not yet supported by our ClientLibrary and might be used as a temporary work-around. The `EHerkenningVerification` object will be added in the future. 

Thank you for opening an issue about this, @SylvesterMachielse 👍

**Comment by ATimmeh33** [comment]
eHerkenning was added in v4.3.0
